install_config/persistent_storage/persistent_storage_ceph_rbd.adoc

```diff
@@ -34,27 +40,71 @@ storage provider.
 ====
 
 [[ceph-provisioning]]
-
 == Provisioning
+
 Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision Ceph volumes in OpenShift, the
-following are required:
+a volume in OpenShift. To provision Ceph volumes in OpenShift, the following
+information is required:
 
-- Existing Ceph volumes, to be defined in the persistent volume object
-- The `*PersistentVolume*` API
+- The ceph key (in base64) to be used as an OpenShift secret object.
```

Comment:
Is base64 coding still supported?
If I do not use base64 does the ceph plugin do the bas64 conversion itself?

----- Original Message -----
> >  
> > -- Existing Ceph volumes, to be defined in the persistent volume object
> > -- The `*PersistentVolume*` API
> > +- The ceph key (in base64) to be used as an OpenShift secret object.
> 
> s/(in base64)//, you don't need to do the encoding yourself anymore.
> 
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/openshift-docs/pull/1732/files/4059fd9b366a0c9b566e993bae7440f1b63b9808#r57236185